File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height; //Declare the variable
    do
    {
        height = get_int("Height?"); //Get the height
    }
    while (!(height >= 1) || !(height <= 8)); //Re ask for the height if value is not between 1 and 8
    {
        for (int i = 1; i <= height; i++)
        {
            for (int j = height - i - 1; j >= 0 ; j--) //Loops for the space
            {
                printf(" ");
            }
            for (int j = 0 ; j < i ; j++) //Loops for the '#'
            {
                printf("#");
            }
            printf("\n");
        }
    }
}

File: mario.py
from cs50 import get_int


def main():
    i = get_positive_int()
    # Manages the display of the pyramid
    for c in range(1, i + 1):
        print((i - c) * " " + c * "#")

# Sanitizes the inputs to get a positive integer


def get_positive_int():
    while True:
        n = get_int("Height: ")
        if n > 0:
            break
    return n


main()
